plugins/modules/vmware_guest_controller.py

```diff
@@ -323,7 +334,7 @@ def create_controller(self, ctl_type, bus_number=0):
             disk_ctl.device = self.scsi_device_type.get(ctl_type)()
             disk_ctl.device.key = -randint(1000, 6999)
             disk_ctl.device.hotAddRemove = True
-            disk_ctl.device.sharedBus = 'noSharing'
+            disk_ctl.device.sharedBus = '%s' % bus_sharing
```

Comment:
```suggestion
            disk_ctl.device.sharedBus = bus_sharing
```

The `bus_sharing` parameter already is a string, you can use it directly here... I think.